SConstruct

```diff
@@ -169,8 +169,9 @@ windows_options = [
         "msvc_version",
         """Version of Visual Studio to use. The default is the newest
            installed version. Specify '12.0' for Visual Studio 2013, '14.0' for
-           Visual Studio 2015, '14.1' ('14.1x') Visual Studio 2017, or '14.2'
-           ('14.2x') for Visual Studio 2019. For version numbers in parentheses,
+           Visual Studio 2015, '14.1' ('14.1x') Visual Studio 2017, '14.2'
+           ('14.2x') for Visual Studio 2019, or '14.3' ('14.3x') for
+           Visual Studio 2022. For version numbers in parentheses,
```

Comment:
I think we should add a note that 12.0 and 14.1 aren't regularly tested; at least that 12.0 isn't regularly tested, since 14.1 will hopefully continue to work as its bracketed by versions we are testing.

Reply:
Probably good to have it in both places

Reply:
I'm honestly hesitant to add this here, as there are several other spots where similar things apply (too many combinations of what may be tested or not). 

It may be better to document this in form of a matrix on the website summarizing what is tested, what should work, and what is expected to fail?